### PR TITLE
New entity creation bugfix

### PR DESCRIPTION
**Pull request by mkysoft**
These changes fix issue #91

### CODE REVIEW

File: generators/entity/index.js [hunk 1]
```diff
@@ -116,7 +116,15 @@ module.exports = class extends BaseGenerator {
           this.log(`\n${chalk.bold.green('I\'m updating the entity for audit ')}${chalk.bold.yellow(this.entityConfig.entityClass)}`);
 
           const entityName = this.entityConfig.entityClass;
-          const jsonObj = this.entityConfig.data;
+          var jsonObj = {};
+          if(this.entityConfig.data === undefined) {
+            jsonObj = {
+                "changelogDate": this.entityConfig.changelogDate,
```
Comment (deepu105) [inline generators/entity/index.js]: no need to quote keys

File: generators/entity/index.js [hunk 2]
```diff
@@ -116,7 +116,15 @@ module.exports = class extends BaseGenerator {
           this.log(`\n${chalk.bold.green('I\'m updating the entity for audit ')}${chalk.bold.yellow(this.entityConfig.entityClass)}`);
 
           const entityName = this.entityConfig.entityClass;
-          const jsonObj = this.entityConfig.data;
+          var jsonObj = {};
```
Comment (deepu105) [inline generators/entity/index.js]: use const instead of var please

File: generators/entity/index.js [hunk 3]
```diff
@@ -116,7 +116,15 @@ module.exports = class extends BaseGenerator {
...
+          if(this.entityConfig.data === undefined) {
```
Comment (deepu105) [inline generators/entity/index.js]: space after if, (please run `npm lint`)

File: generators/entity/index.js (hunk 1 above)
Comment (mkysoft) [reply generators/entity/index.js]: I removed them.

File: generators/entity/index.js (hunk 2 above)
Comment (mkysoft) [reply generators/entity/index.js]: I changed it to const with inline if.

File: generators/entity/index.js (hunk 3 above)
Comment (mkysoft) [reply generators/entity/index.js]: I install eslint to vs code and check, thanks.

**Comment by deepu105** [comment]
@pascalgrimaud the builds are failing with a weird error, any idea?

**Comment by pascalgrimaud** [comment]
Quick answer would be to upgrade to Node 8.11.2.
Not sure it works well with Node 6, as it's deprecated

**Comment by mkysoft** [comment]
Do you plan accept this request?

**Pull request merged**